pingouin/tests/test_parametric.py

```diff
@@ -181,6 +181,16 @@ def test_anova(self):
         aov = anova(dv='Pain threshold', between='Hair color', data=df_pain,
                     effsize="n2", detailed=True).round(3)
         assert aov.at[0, 'n2'] == .576
+        # Compare residuals with R
+        # Use different dataset to produce residuals more than 3 decimal places
+        # aov(Cholesterol ~ Risk, DF)$residuals
+        resid = read_dataset('anova3').anova(
+            dv='Cholesterol', between=['Risk']
+        ).residuals_
+        array_equal(
+            resid[0:5].round(3),
```

Comment:
I noticed that you are always testing only the first 5 values of the residuals. We need to test all values, i.e. array_equal on all the residuals